docs/about/contributing.mdx

```diff
@@ -126,3 +126,44 @@ We're deeply thankful for the backing from the Open Collective, Open Source Coll
     By covering our hosting costs, Vercel directly contributes to Meshtastic's growth, enhancing our web scalability and presence. Their sponsorship is crucial for our continued development and focus on creating a robust platform for decentralized communication.
   </div>
 </div>
+
+## Contributing to the Meshtastic Blog
+
+### Purpose
+
+To share knowledge, updates, and innovations related to Meshtastic helping to enrich, engage, and grow the community.
```

Comment:
How about: "To foster growth and collaboration by sharing knowledge, updates, and innovations from the Meshtastic community."